lua/nvim-treesitter/textobjects/attach.lua

```diff
@@ -10,6 +10,10 @@ function M.make_attach(normal_mode_functions, submodule)
     lang = lang or parsers.get_buf_lang(bufnr)
 
     for _, function_call in pairs(normal_mode_functions) do
+      local description = function_call:gsub("_", " ")
+      if type(function_call) == "table" then
```

Comment:
Should this type check happen before the gsub call? If the function is not a string but a table I expect this to fail.

Reply:
You're right! Thank you!